platforms/posix/CMakeLists.txt

```diff
@@ -73,20 +85,12 @@ elseif ("${BOARD}" STREQUAL "bbblue")
 	target_link_libraries(px4 PRIVATE robotcontrol)
 
 	add_custom_target(upload
-		COMMAND scp -r ${PX4_SOURCE_DIR}/posix-configs/bbblue/*.config  debian@BBBluePX4:/home/debian/px4/posix-configs
-		COMMAND scp -r ${PX4_SOURCE_DIR}/ROMFS   $<TARGET_FILE:px4>     debian@BBBluePX4:/home/debian/px4
+		COMMAND rsync -arh --progress ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${PX4_SOURCE_DIR}/posix-configs/bblue/*.config ${PX4_SOURCE_DIR}/ROMFS debian@BBBluePX4:/home/debian/px4
```

Comment:
@UAV-Pilot This changes the uploading for bbblue: rsync is faster and the directory structure is a bit different. Can you test it please?
You need to run px4 like this:
```
cd px4
./bin/px4 -s px4.config
```

Reply:
After changing "posix-configs/bblue/" to "posix-configs/bbblue/", uploading with rsync worked as expected. Also note that rsync did not ask for password of the user on BeagleBone Blue board probably due to prior SSH authorized_keys setup.

Verified that without explicitly putting px4-alias.sh in the PATH, px4-alias.sh was executed in all of the following 3 scenarios (although it might not be necessary in daemon mode):
1. manual start: sudo ./bin/px4 -s px4.config  (librobotcontrol currently requires root)
2. auto start via /etc/rc.local
3. auto start via systemd service

Reply:
Nice, thanks for the quick and extensive testing! I'll fix up the path and have a look at the daemon mode.